src/server/pps/server/api_server.go

```diff
@@ -173,6 +179,19 @@ func validateNames(names map[string]bool, input *pps.Input) error {
 	return nil
 }
 
+func validateName(name string) error {
+	if name == "" {
+		return errors.Errorf("input must specify a name")
+	}
+	switch name {
+	case common.OutputPrefix:
```

Comment:
I think this can be `case common.OutputPrefix, common.EnvFileName:` and avoid the fallthrough.